src/elm/Page/Community.elm

```diff
@@ -482,6 +484,51 @@ viewAction loggedIn metadata maybeDate action =
             ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Html Msg
+viewClaimModal loggedIn model =
+    case model.modalStatus of
+        Opened isLoading actionId ->
+            let
+                t s =
+                    I18Next.t loggedIn.shared.translations
+                        s
+
+                text_ s =
+                    text (t s)
+            in
+            div []
+                [ Modal.initWith { closeMsg = CloseClaimConfirmation, isVisible = True }
+                    |> Modal.withHeader (t "claim.modal.title")
+                    |> Modal.withBody [ text_ "dashboard.check_claim.body" ]
+                    |> Modal.withFooter
+                        [ button
+                            [ class "flex-1 block button button-secondary mb-4 button-lg w-full md:w-40 md:mb-0"
+                            , if not isLoading then
+                                onClick CloseClaimConfirmation
+
+                              else
+                                onClick NoOp
+                            , disabled isLoading
+                            ]
+                            [ text_ "dashboard.check_claim.no" ]
+                        , button
+                            [ class "flex-1 block button button-primary button-lg w-full md:w-40"
```

Comment:
There are predefined classes for this kind of buttons from the modal's footer: `modal-cancel` and `modal-accept`. Please, see [`src/elm/Page/Dashboard.elm:225`](https://github.com/cambiatus/frontend/blob/master/src/elm/Page/Dashboard.elm#L225) for the reference (the claim popup on the Dashboard).

Reply:
Nice! thanks for the review and pointing out an example! It helped a lot 💪